### PR TITLE
dav1d: update to 1.0.0

### PR DESCRIPTION
**Pull request by mustaqimM**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- [x] I built this PR locally for my native architecture, `x86_64 libc`

The shlib should have been changed in the last version to libdav1d5. Also, why does `libdav1d3` still show up when querying?

### CODE REVIEW

File: common/shlibs [hunk 1]
```diff
@@ -3557,7 +3557,7 @@ libcsfml-window.so.2.5 CSFML-2.5_1
 libcsfml-system.so.2.5 CSFML-2.5_1
 libcsfml-network.so.2.5 CSFML-2.5_1
 libtokyocabinet.so.9 tokyocabinet-1.4.48_1
-libdav1d.so.5 libdav1d4-0.8.1_1
+libdav1d.so.5 libdav1d5-0.8.1_1
```
Comment (leahneukirchen) [inline common/shlibs:3560]: Wrong version.

**Review by leahneukirchen — CHANGES_REQUESTED** [review]

File: common/shlibs (hunk 1 above)
Comment (mustaqimM) [reply common/shlibs:3560]: It should have been changed in the last update to `libdav1d5` to match the `so` version. Should I just leave it as is?

Comment (leahneukirchen) [reply common/shlibs:3560]: No, you should put 0.9.2_1 there.

**Comment by 38github** [comment]
What is needed to get this into the repository?

**Comment by leahneukirchen** [comment]
@mustaqimM please rebase, then this is good to go.

**Comment by 38github** [comment]
> @mustaqimM please rebase, then this is good to go.

Can you please explain what a rebase means, please? I am a novice.

**Pull request merged**